script/reviewer.go

```diff
@@ -19,11 +20,84 @@ func (r *Reviewer) Review() {
 
 	r.application.Parse(r.gitHub.Issue)
 
-	if isTestingIssue() {
-		if r.application.IsValid() {
-			debugMessage("Application has no problems")
-		} else {
-			debugMessage("Application problems:", r.application.RenderProblems())
+	status := r.getStatus()
+	r.updateLabels(status)
+	r.createComment(status)
+}
+
+func (r *Reviewer) getStatus() string {
```

Comment:
[idiomatic] I'd encourage you to create a `Status` enum and have this method return enum values

```go
type Status int

const (
    Closed Status = iota
    Approved
    Reviewing
    Invalid
    New
)
```

It doesn't look like you need the string representation of these, but you can add a `String()` method onto the type with:

```go
func (s Status) String() string {
    switch s {
    case Closed:
        return "closed"
    ...
}
```

Reply:
Don't think we need string reps 👍